### PR TITLE
*: update tikv/client-go and fix stale read metrics in grafana

### LINKED ISSUE #45339: refine stale read metrics

### PR DESCRIPTION
**Pull request by crazycs520**
### What problem does this PR solve?

Issue Number: ref #45339 #45297

Problem Summary: update tikv/client-go and add stale read metrics in grafana

related PR: https://github.com/tikv/client-go/pull/886

### What is changed and how it works?

Add stale read metrics as following show:

image

### Check List

Tests 

- [ ] Unit test
- [ ] Integration test
- [ ] Manual test (add detailed scripts or steps below)
- [ ] No code

Side effects

- [ ] Performance regression: Consumes more CPU
- [ ] Performance regression: Consumes more Memory
- [ ] Breaking backward compatibility

Documentation

- [ ] Affects user behaviors
- [ ] Contains syntax changes
- [ ] Contains variable changes
- [ ] Contains experimental features
- [ ] Changes MySQL compatibility

### Release note

Please refer to [Release Notes Language Style Guide](https://pingcap.github.io/tidb-dev-guide/contribute-to-tidb/release-notes-style-guide.html) to write a quality release note.

```release-note
None
```

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -94,7 +94,7 @@ require (
 	github.com/stretchr/testify v1.8.2
```
Comment (hawkingrei) [inline go.mod:94]: Please run ```make bazel_prepare```.

**Review by you06 — APPROVED** [review]

**Review by ekexium — DISMISSED** [review]

File: metrics/grafana/tidb.json [hunk 2]
```diff
@@ -10200,6 +10200,315 @@
...
+              "format": "bits",
+              "label": null,
+              "logBase": 1,
+              "max": null,
+              "min": null,
+              "show": true
+            },
+            {
+              "$$hashKey": "object:328",
+              "format": "short",
```
Comment (ekexium) [inline metrics/grafana/tidb.json:10500]: Is this supposed to be something like "Bps"?

File: metrics/grafana/tidb.json [hunk 3]
```diff
@@ -10200,6 +10200,315 @@
...
+            "buckets": null,
+            "mode": "time",
+            "name": null,
+            "show": true,
+            "values": []
+          },
+          "yaxes": [
+            {
+              "$$hashKey": "object:327",
+              "format": "bits",
```
Comment (ekexium) [inline metrics/grafana/tidb.json:10491]: Is this supposed to be something like "Bps"?

Comment (crazycs520) [reply metrics/grafana/tidb.json]: Nice catch, addressed.

Comment (crazycs520) [reply metrics/grafana/tidb.json]: image

Comment (crazycs520) [reply metrics/grafana/tidb.json]: image

**Review by ekexium — APPROVED** [review]

**Pull request merged**